### ISSUE TITLE
Is is possible to specify a currently existing CodeStar connection ARN when using Bitbucket as the Source?

### ISSUE THREAD

**Issue by msperrin** [aws/copilot-cli#2012]
Currently the pipeline init flow seems to be setup so that a new CodeStar connection is created. It would be useful to be able to use a pre existing connection (pass the ARN as a flag?) as the Bitbucket authentication process requires a level of privilege that is not available within our org by default, and we have pre-existing connections already setup.

**Comment by efekarakus** [comment]
That makes sense, thanks for the issue! We can add an additional field in our `pipeline.yml` file to to use an existing connection name from the account and region where the pipeline is going to be created to skip the connection creation:
```yaml
source:
  provider: 'Bitbucket'
  connection_name: 'aEXAMPLE-8aad-4d5d-8878-dfcab0bc441f'
```

**Comment by huanjani** [comment]
Hello, @msperrin!

Yes, it's possible, and a great idea! Thanks for the feedback--it's good to know how this functionality is being used.
When we designed the CodeStar Connection workflow, we wanted to take the heavy lifting off of the user. But at the same time, we want to enable customization where necessary. 

Until we release the change @efekarakus mentions ☝🏼 , here is a temporary workaround for using your existing connection. Unfortunately, these changes won't persist when you run `copilot pipeline update` because a new template will be generated.
1. Go into the CloudFormation page of the AWS console.
2. Choose the Stack for the pipeline and select 'Update', 'Edit template in designer', 'View in Designer'
3. There are three places where you can change the automatically created Connection (`SourceConnection`) to your pre-existing Connection ARN:

A.
```
    - codestar-connections:PassConnection
    - codestar-connections:PassedToService
  Resource:
    - [HERE]
- Effect: Allow
  Action:
```
B.
```
  Provider: CodeStarSourceConnection
Configuration:
  ConnectionArn: [HERE]
```
C.
```
Outputs:
  PipelineConnectionARN:
    Description: "ARN of CodeStar Connections connection"
    Value: [HERE]
```

4. Save your edits by clicking `create the stack' (cloud with up arrow) and clicking through to update. 
5. From the CodePipeline console page, click 'Release Change'.

**Comment by efekarakus** [comment]
This is now released in v1.5.0! https://github.com/aws/copilot-cli/releases/tag/v1.5.0 🚀
